lua/telekasten.lua

```diff
@@ -1431,11 +1440,12 @@ local function RenameNote()
     newname = newname:gsub("[" .. M.Cfg.extension .. "]+$", "")
     local newpath = newname:match("(.*/)")
 
+    -- If no subdir specified, place the new note in the same place as old note
```

Comment:
I am a bit wary here. IMHO the "new" note should always go into the same directory as the existing one, since it's a rename...?

Reply:
Let's say I have a very hierarchical zettelkasten. One note is in `philosophy/mynote.md`, but I added stuff and now I feel that the note needs to be in `culture/newname.md`. If I give it `culture/newname.md`, the note will be moved to the proper directory (dir may need to be created, I still have to do that). If I only give it `newname`, and I have `subdirs_in_links=true`, the note will be placed in `philosophy/newname.md`.

This just gives liberty to the user to move notes around subdirs from within telekasten, while taking care of updating links.

Reply:
OK, I agree 😊